src/Chapter2.hs

```diff
@@ -177,31 +177,31 @@ Evaluate the following expressions in GHCi and insert the answers. Try
 to guess first, what you will see.
 
 >>> [10, 2] ++ [3, 1, 5]
-
+[10, 2, 3, 1, 5]
```

Comment:
Doctest expects strings in the exact same format as GHCi produces. And for lists, GHCi doesn't add spaces. So, to make the doctest pass, you need to remove extra spaces after commas.
```suggestion
[10,2,3,1,5]
```